cmd/program/program.go

```diff
@@ -216,6 +279,10 @@ func (p *Project) CreateFileWithInjection(pathToCreate string, projectPath strin
 	case "routes":
 		createdTemplate := template.Must(template.New(fileName).Parse(string(p.FrameworkMap[p.ProjectType].templater.Routes())))
 		err = createdTemplate.Execute(createdFile, p)
+
+	case "services":
+		createdTemplate := template.Must(template.New(fileName).Parse(string(p.DBDriverMap[p.DBDriver].templater.Service())))
```

Comment:
The current set up creates this in a 
```
root/services
```
folder.

I am wondering to keep project structure, can we add it to the existing internal directory and create a new sub-folder called database?
```
/internal/database/database.go
```